### PR TITLE
automatically handle usbmuxd and require root for linux

### PR DESCRIPTION
**Pull request by alex-free**

### CODE REVIEW

File: palera1n.sh [hunk 1]
```diff
@@ -1,5 +1,17 @@
 #!/usr/bin/env bash
 
+os=$(uname)
+
+if [ "$os" = 'Linux' ]; then
+    if [ "$EUID" != 0 ]; then
+        echo "Error: Palera1n requires root privileges on Linux to connect to your device, please execute palera1n with sudo or as root."
+        exit 1
+    fi
+    systemctl stop ubsmuxd > /dev/null 2>&1
```
Comment (0xallie) [inline palera1n.sh:10]:
```suggestion
    systemctl stop usbmuxd > /dev/null 2>&1
```

File: palera1n.sh [hunk 2]
```diff
@@ -1114,3 +1125,8 @@ echo "Enjoy!"
 } 2>&1 | tee logs/${log}
 
 popd &> /dev/null
+
+if [ "$os" = 'Linux' ]; then
+    killall usbmuxd > /dev/null 2>&1
+    systemctl start usbmuxd > /dev/null 2>&1
+fi
```
Comment (0xallie) [inline palera1n.sh:1132]: Add this in `_exit_handler` instead

File: palera1n.sh (hunk 1 above)
Comment (alex-free) [reply palera1n.sh]: Fixed typo in force push, oops.

File: palera1n.sh (hunk 2 above)
Comment (alex-free) [reply palera1n.sh]: Done

**Pull request closed without merging**